Docs/user_guide/modules/retiredmodules.md

```diff
@@ -0,0 +1,12 @@
+The modules listed below have been retired and are no longer present in 3D Slicer. This page exists as a convenient marker for historical details about retired modules.
+
+Retiring modules reduces burden on slicer developers.
+
+A short list of reasons why a module may have been retired
+
+- The module has been superseded with another, and is not recommended
+- The module has no development support, and no longer runs properly
```

Comment:
The list does not seem to be rendered properly:

![image](https://user-images.githubusercontent.com/219043/143253404-0681a066-30be-4445-aa0a-acabf7491e6f.png)

Reply:
I noticed that as well. Not sure why. I'll push again to see if something changes.

Reply:
With latest push they are still not rendering appropriately. Not sure why